01PITT_INST-ULS_1_5/js/custom.js

```diff
@@ -308,4 +317,89 @@ angular.module('getTempAddress', []).controller('prmRequestAfterController',func
 }).component('prmRequestAfter',{
   controller:'prmRequestAfterController'
 });
+
+
+//SMS Notifications User Preferences Interface
+angular.module('smsNotifications', []).controller('smsNotificationsController',function($scope,$http){
+	this.$onInit = function () {
+		$scope.smsNumber='';
+		//jwt is stored with double quotes?
+		var jwt=this.primoExplore.storageutil.sessionStorage.primoExploreJwt.slice(1, -1);
+
+		//SMS STATUS
+		$http({
+			method : "GET",
+			url : "https://dev-patron.libraries.pitt.edu/address/sms/?jwt="+jwt,
```

Comment:
I can't find one.  We could go off of the URL, but this breaks testing via localhost.

Reply:
How about:
```
"https://"+( window.location.hostname == 'pitt.primo.exlibrisgroup.com' ? '' : 'dev-')+"patron.libraries.pitt.edu/address/sms/?jwt="+jwt
```
?

Reply:
Also, is there any way to abstract this as a "setting" of some form (even if just a dedicated javascript global), so that others who might want to use this module just need to change a setting, rather editing deep within the js?

Reply:
I like this.  Since local testing and the sandbox both rely on dev-patron, we can lump those together if the url is for our production site.  Seems to work fine.  6ef2e08a4ae5ac13d3d3b63a13db25cc0e0a1cb4